### ISSUE TITLE
[speech-to-text] Documentation of TranscriptionSettings in speech-to-text points to a dead link

### ISSUE THREAD

**Issue by clindsay** [watson-developer-cloud/ios-sdk#422]
Documentation of TranscriptionSettings in SpeechToTextV1 contains this line:
```
/// sampled. If `nil`, then the default model will be used. Visit https://ibm.biz/BdH93p
/// for more information about the available models.
```
However, that link leads to a 404 page.

Xcode 7.3.1, ios-sdk 0.6.1, iOS Version 9.0

**Comment by glennrfisher** [comment]
Thanks for letting us know! We'll be sure to fix that in an update as soon as possible.

Here's an updated link to the [models](https://www.ibm.com/watson/developercloud/doc/speech-to-text/input.shtml#models).

**Comment by clindsay** [comment]
Thanks for the quick response! =D

**Comment by glennrfisher** [comment]
This is fixed with PR #424.
